tests/Hamcrest/File/IsFileWithSizeTest.php

```diff
@@ -0,0 +1,204 @@
+<?php
+
+/**
+ * @author Vasek Brychta <vaclav@brychtovi.cz>
+ */
+
+namespace Hamcrest\File;
+
+use Hamcrest\AbstractMatcherTest;
+
+class IsFileWithSizeTest extends AbstractMatcherTest
+{
+	private static $FILE_PATH;
+	private static $FILE;
+	private static $ACTUAL_SIZE;
+
+	private static $EMPTY_FILE;
+
+	/**
+	 * @beforeClass
+	 */
+	public static function initializeFiles()
+	{
+		self::$FILE_PATH = __FILE__;
+		self::$FILE = new \SplFileInfo(self::$FILE_PATH);
+		self::$ACTUAL_SIZE = filesize(self::$FILE_PATH);
+
+		self::$EMPTY_FILE = new class('/tmp/empty') extends \SplFileInfo {
+			public function getSize()
+			{
+				return 0;
+			}
+		};
+	}
+
+	protected function createMatcher()
+	{
+		return IsFileWithSize::aFileWithSize(not(anything()));
+	}
+
+	public function testMatchesAFileWithEqualSize()
+	{
+		$this->assertMatches(
+			aFileWithSize(self::$ACTUAL_SIZE),
```

Comment:
Not sure, that it's a good idea to use matcher functions from a global namespace vs namespaced methods.

Please see how it's implemented in other matchers.